src/Buffer.mo

```diff
@@ -735,6 +760,16 @@ module {
 
   /// Returns a copy of `buffer`, with the same capacity.
   ///
+  ///
+  /// Example:
+  /// ```motoko include=initialize
+  ///
+  /// let buffer = Buffer.Buffer<Nat>(2);
+  /// buffer.add(1);
+  /// 
+  /// Buffer.clone(buffer);
```

Comment:
```suggestion
  /// let clone = Buffer.clone(buffer);
  /// Buffer.toArray(clone);
```

The interpreter won't display the elements in the buffer if you evaluate it. If you convert it to an array, the user can see the elements